tasks.py

```diff
@@ -43,21 +43,26 @@ def imageid(ctx):
         format='{{.Id}}'
     ))
 
+
 def link_phutil_map(ctx):
-    ctx.run("ln -fs `pwd`/phutil_map /tmp/phutil_map")
+    """Create link to phutil_map"""
+    ctx.run("cp `pwd`/phutil_map /tmp/phutil_map")
+
```

Comment:
@mars-f had a different idea where we share the subdirectory instead of the direct file, so no weak link is required - https://github.com/mozilla-conduit/phabricator-extensions/pull/16#issuecomment-305841652